x-pack/plugins/alerting/server/saved_objects/migrations.test.ts

```diff
@@ -2011,6 +2011,18 @@ describe('successful migrations', () => {
       const alert = getMockData({}, true);
       expect(migration800(alert, migrationContext)).toEqual(alert);
     });
+
+    test('add threatIndicatorPath default value to threat match rules', () => {
```

Comment:
Can we add a test to make sure nothing changes if `threatIndicatorPath` is already populated?